LICENSE

```diff
@@ -1,6 +1,9 @@
-Copyright (c) 2001, 2011 NoMachine - http://www.nomachine.com/.
+Copyright (c) 2001, 2011 NoMachine (http://www.nomachine.com)
+Copyright (c) 2011-2016 Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
+Copyright (c) 2015-2016 Qindel Group (http://www.qindel.com)
 
-NX-X11 and NX extensions to X are copyright of NoMachine.
+nx-X11, NX extensions to X (nxcompext, nxcompshad), nxagent and nxproxy
+are copyright of the aforementioned copyright holders.
 
```

Comment:
Hi,

On Tue Jul 5 13:54:51 2016 GMT+0200, Ulrich Sibiller wrote:
> In LICENSE:
> >  
> > -NX-X11 and NX extensions to X are copyright of NoMachine.
> > +nx-X11, NX extensions to X (nxcompext, nxcompshad), nxagent and nxproxy
> > +are copyright of the aforementioned copyright holders.
> >  
> What are the criteria to be listed or not? What about all contributors? 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view itd.￼
In theory anyone contributing can be listed as persons. I think we should add you and Ionic and Alex from X2Go, at least.

Please let me know how you want to be mentioned.

@Ionic: dito.

Mike

-- 
Sent from my Jolla